rules/windows/persistence_suspicious_image_load_scheduled_task_ms_office.toml

```diff
@@ -19,9 +44,70 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*", "endgame-*"
 language = "eql"
 license = "Elastic License v2"
 name = "Suspicious Image Load (taskschd.dll) from MS Office"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating Suspicious Image Load (taskschd.dll) from MS Office
+
+Microsoft Office is a suite of applications designed to help with productivity and completing common tasks on a computer. You can create and edit documents containing text and images, work with data in spreadsheets and databases, and create presentations and posters. As it is some of the most-used software across companies, MS Office is frequently targeted for initial access. It also has a wide variety of capabilities that attackers can take advantage of.
+
+`taskschd.dll` provides Command Object Model (COM) interfaces for the Windows Task Scheduler service, allowing developers to programmatically manage scheduled tasks.
+
+This rule looks for a MS Office process loading `taskschd.dll`, which may indicate an adversary abusing COM to configure a scheduled task. This can be the result of a phishing attack, where an malicious office document is registering the scheduled task to download the malware "stage 2" or to establish an persistent access to the attacker.
```

Comment:
```suggestion
This rule looks for an MS Office process loading `taskschd.dll`, which may indicate an adversary abusing COM to configure a scheduled task. This can be the result of a phishing attack, where a malicious office document is registering the scheduled task to download the malware "stage 2" or to establish persistent access to the attacker.
```